VTEX - Pick and Pack API.json

```diff
@@ -0,0 +1,4196 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+        "title": "Pick and Pack API",
+        "description": "The Pick and Pack API provides comprehensive functionality for order management, picking operations, shipment tracking, and returns processing. The main capabilities include:\n\n- Retrieving detailed order information including items, packages, and status\n\n- Accessing worksheet data for picking operations\n\n- Querying shipment tracking and delivery details\n\n- Getting facility information",
+        "contact": {},
+        "version": "1.0"
+    },
+    "servers": [
+        {
+            "url": "https://auth.pickingnpacking.com/prod/{environment}",
+            "variables": {
+                "environment": {
+                    "default": "DefaultParameterValue",

```

Comment:
This is too generic. I think even something like "environment" would be better. But maybe the team can provide something more specific/realist, like we have "vtexcommercestable" for core commercer APIs. Did you check it with them?